### PR TITLE
Strip scheme

### PR DESCRIPTION
**Pull request by jcollins-g**
This mostly works, except somehow I broke the crossdart test.

### CODE REVIEW

**Comment by devoncarew** [comment]
From this failure: https://travis-ci.org/dart-lang/dartdoc/jobs/184374678#L550 it does look like we're stripping the hyperlinking off now. Will take a quick look at the code.

File: pubspec.lock [hunk 1]
```diff
@@ -306,6 +306,6 @@ packages:
       name: yaml
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.10"
+    version: "2.1.12"
 sdks:
-  dart: ">=1.17.0-dev.6.2 <2.0.0"
+  dart: ">=1.21.0-dev.2.0 <2.0.0"
```
Comment (jcollins-g) [inline pubspec.lock]: This is a generated file.  How can I make sure pub doesn't undo this if I force-override it manually?

Comment (devoncarew) [inline pubspec.lock]: Hmm, good question; I didn't look closely enough and assumed this was the pubspec.yaml file.

It looks like what you really need is a slight bump to the markdown package? In which case, you could revert the changes to this file, run `pub get`, and it'll jump pull down the min. necessary to satisfy the changed deps.

You probably ran `pub upgrade`, which is more of a move-everything-to-the-latest operation.

**Comment by devoncarew** [comment]
@jcollins-g, progress on this? From IMing it sounds like the crossdart test is not hermetic or is racy?

**Comment by jcollins-g** [comment]
Fixed it.  Short version was the offset wasn't computed right, and I'm still getting used to IntelliJ and the debugger.  I've improved the test now so that it won't require by-hand adjustments of the offset every time, PTAL.

File: test/model_test.dart [hunk 2]
```diff
@@ -1010,14 +1010,20 @@ String topLevelFunction(int param1, bool param2, Cool coolBeans,
...
+          .readAsStringSync()
+          .indexOf('Map<X, Y> convertToMap');
+      expect(offset, isNonNegative,
+             reason: "Can't find convertToMap function in ${fakePath}");
       if (Platform.isWindows) fakePath = fakePath.replaceAll('/', r'\\');
       new File(p.join(Directory.current.path, "crossdart.json"))
           .writeAsStringSync("""
               {"$fakePath":
-                {"references":[{"offset":1069,"end":1072,"remotePath":"http://www.example.com/fake.dart"}]}}
+                {"references":[{"offset":${offset},"end":${offset+3},"remotePath":"http://www.example.com/fake.dart"}]}}
```
Comment (devoncarew) [inline test/model_test.dart]: good find - this should future proof this test

**Comment by devoncarew** [comment]
Just to minimize diffs, can you revert to the original version of the pubspec.lock file, keep the change to the pubspec.yaml file, and run `pub get`? It should update the packages in the lock file just enough to capture the markdown change (and any newer versions of dependent packages as necessary).

**Comment by jcollins-g** [comment]
@devoncarew Done.

**Comment by devoncarew** [comment]
lgtm!

Looks like a recent roll of the dev sdk may have changed the generated output - I'll take a look.

**Comment by jcollins-g** [comment]
Forgot to write here that this fixes #1147.

**Pull request merged**